tests/conftest.py

```diff
@@ -66,6 +66,7 @@ def restore_env_variables():
         "RANK",  # set by DeepSpeed
         "POPLAR_ENGINE_OPTIONS",  # set by IPUPlugin
         # set by XLA
+        "TF2_BEHAVIOR",
```

Comment:
Same comment as https://github.com/PyTorchLightning/pytorch-lightning/pull/9722#discussion_r717529207

It must be coming from `TensorBoard`, a better solution would be to disable the logger for that test

Reply:
It seems coming from multiple tests. Let's address this within another PR.